### ISSUE TITLE
Request：add "move speed","Fly hack"

### ISSUE THREAD

**Issue by xuewuji** [Akebi-Group/Akebi-GC#80]
move speed : Just affect Player‘s “Run/Fly/Swim Speed”，not “no clip” or “no collision”，just move more distances in the same time.

fly hack : Player can stay in the air like stay in the land， will not fall down.

**Comment by G1DE30N** [comment]
Move speed not needed because game have speed hack prediction itself - If you move faster then normal game simple teleports you back. You can try it with noclip and proofed by yourself.

Fly hack not needed because noclip, why just not to stay in air in them?

**Comment by RyujinZX** [comment]
teleports you back - This can be turned off =)

**Comment by G1DE30N** [comment]
> teleports you back - This can be turned off =)

Yep by changed speed to normal (not much different from the standard) values :D

**Comment by RyujinZX** [comment]
> > teleports you back - This can be turned off =)
> 
> Yep by changed speed to normal (not much different from the standard) values :D Okay maybe move speed can be implemented in packed mode like inf stamina. But i think this thing maybe slightly detecteable because too much packet you send in unit of time

What if I told you that I could run at high speed and I wouldn't be teleported back? ))

**Comment by G1DE30N** [comment]
> > > teleports you back - This can be turned off =)
> > 
> > 
> > Yep by changed speed to normal (not much different from the standard) values :D Okay maybe move speed can be implemented in packed mode like inf stamina. But i think this thing maybe slightly detecteable because too much packet you send in unit of time
> 
> What if I told you that I could run at high speed and I wouldn't be teleported back? ))

Alright, intrigued, how?

**Comment by RyujinZX** [comment]
We just did not add this feature :)

**Comment by G1DE30N** [comment]
> We just did not add this feature :)

Alright got it. Another question, why this needed? (Again because noclip alredy exists)

**Comment by RyujinZX** [comment]
> > We just did not add this feature :)
> 
> Alright got it. Another question, why this needed? (Again because noclip alredy exists)

To use noclip 100, and not to teleport back

**Comment by G1DE30N** [comment]
> > > We just did not add this feature :)
> > 
> > 
> > Alright got it. Another question, why this needed? (Again because noclip alredy exists)
> 
> To use noclip 100, and not to teleport back

Alright. No more questions. Maybe this 'thing' needed to somebody :D

**Comment by xuewuji** [comment]
> Move speed not needed because game have speed hack prediction itself - If you move faster then normal game simple teleports you back. You can try it with noclip and proofed by yourself.
> 
> Fly hack not needed because noclip, why just not to stay in air in them?

When you use 1.5x or 2x speed, you will not be teleports.
i think noclip is unsafe,fly hack just keep you in the air forever，so you can fly to anywhere form a high mountain.

**Comment by G1DE30N** [comment]
> > Move speed not needed because game have speed hack prediction itself - If you move faster then normal game simple teleports you back. You can try it with noclip and proofed by yourself.
> > Fly hack not needed because noclip, why just not to stay in air in them?
> 
> When you use 1.5x or 2x speed, you will not be teleports. i think noclip is unsafe,fly hack just keep you in the air forever，so you can fly to anywhere form a high mountain.

Noclip is safe in speeds 5.5 and above - Month of active using by me proffed it

**Comment by tedzify** [comment]
> Alright got it. Another question, why this needed? (Again because noclip alredy exists)

Maybe some people just want to run faster without clipping into the ground.

**Comment by DarkXonline** [comment]
> > Alright got it. Another question, why this needed? (Again because noclip alredy exists)
> 
> Maybe some people just want to run faster without clipping into the ground.

I second this. I really want a simple "I move fast, I fly fast, but I don't want to get into the ground" kinda thing

**Comment by fid3n** [comment]
noclip speed 10

**Comment by CallowBlack** [comment]
We don't plan to add `Fly hack` because there is `No clip`.

**Comment by Shichiha** [comment]
> but I don't want to get into the ground" kinda thing

If you didn't know, there is a noclip "Velocity" mode which you can turn on.

It changes your velocity instead of position, try tweaking the settings a bit.
